recipes-ros/mavlink/mavlink_2016.5.20.bb

```diff
@@ -0,0 +1,26 @@
+#
+# MAVLink message marshaling library.
+# This package provides C-headers and pymavlink library
+# for both 1.0 and 2.0 versions of protocol.
+#
+
+DESCRIPTION = "MAVLink message marshaling library"
+DEPENDS = ""
+LICENSE = "LGPLv3"
```

Comment:
That's the License of the Mavlink Project. It should be independent of the recipe license (MIT since no other is specified)